gdc_client/upload/client.py

```diff
@@ -38,11 +37,6 @@
 
     from multiprocessing.pool import ThreadPool as Pool
```

Comment:
I think this can be safely removed.

This might make sense for a different investigation ticket/task. We might be able to safely remove the `freeze_support()` call since we switched to `concurrent.futures.ThreadPoolExecutor` which does not use the `multiprocessing` I believe. The `concurrent.futures.ProcessPoolExecutor` uses the `multiprocessing` module so a call to `freeze_support()` on windows would still be needed.